index.bs

```diff
@@ -346,35 +346,48 @@ limiting the feature to secure contexts might cause problems
 for code (such as libraries) that may be used in either secure or non-secure contexts.
 </div>
 
-<h3 id="do-not-expose-use-of-private-browsing-mode">Do not reveal that private browsing mode is engaged</h3>
+<h3 id="do-not-expose-use-of-private-browsing-mode">Don't reveal that private browsing mode is engaged</h3>
+
+Make sure that your feature doesn't give authors a way to detect private browsing mode.
 
 Some people use [private browsing mode](https://www.w3.org/2001/tag/doc/private-browsing-modes/)
-for their own <a href="#safe-to-browse">safety</a>;
-just the fact that they have or are using private browsing mode
+for their own <a href="#safe-to-browse">safety</a>.
+Because of this, the fact that someone is using private browsing mode
 may be [sensitive information](https://www.w3.org/TR/security-privacy-questionnaire/#sensitive-data) about them.
-[Consider the harm](https://www.w3.org/2001/tag/doc/ethical-web-principles/#noharm) that could be visited upon people—especially vulnerable people—were
-their use of private browsing mode revealed
-to others who wield power over them
-(such as employers,
-parents,
-partners,
+This information may [harm](https://www.w3.org/2001/tag/doc/ethical-web-principles/#noharm) people
+if it is revealed to a web site contolled by others who have power over them
+(such as employers, parents, partners,
 or [state actors](https://www.w3.org/2001/tag/doc/ethical-web-principles/#expression)).
 
 Given such dangers,
 websites should not be able to detect that private browsing mode is engaged.
 
 <div class="example">
-
 User Agents which support [IndexedDB](https://www.w3.org/TR/IndexedDB/)
 should not disable it in private browsing mode,
 because that would reveal that private browsing mode is engaged
+</div>
+
+<div class="example">
+The [Payment Request API](https://www.w3.org/TR/payment-request/)'s
+[show() method](https://www.w3.org/TR/payment-request/#show-method),
+when called,
+allows User Agents <q cite="https://www.w3.org/TR/payment-request/#show-method">to act
+as if the user had immediately
+[aborted the payment request](https://www.w3.org/TR/payment-request/#dfn-user-aborts-the-payment-request)</q>.
```

Comment:
It's not clear to me that this is a good example.  Are there other cases where the UA will act as though the user had immediately aborted the payment request, or is that a clear sign the user is in private mode?  I think if you want to explain why this is a good example, you should explain what other things would appear the same to the UA.

Reply:
Hm, I just moved this up from the other private browsing section, since it seemed to fit better here.

I think @hober might have written the original example?

Reply:
I think it fits better in the other section (where it was originally) because I think it might actually be a bad example for this one... although I'm not sure, and it's possible it could be a good example here with more detail added to demonstrate that.

Reply:
@hober Could use your opinion here as well

Reply:
I think this is a pre-existing problem with this example that is made a bit more obvious by the move. We should improve the example in a follow-on PR. I think @alice is filing an issue for this.